packages/ffe-form-react/src/PhoneNumber.js

```diff
@@ -18,14 +20,42 @@ export default class PhoneNumber extends React.Component {
             onCountryCodeBlur,
             onNumberBlur,
             disabled,
-            countryCodeInvalid,
-            numberInvalid,
+            countryCodeFieldMessage,
+            numberFieldMessage,
+            countryCodeAndNumberFieldMessage,
             className,
+            extraMargin = true,
             dark,
             countryCodeRef,
             numberRef,
         } = this.props;
 
+        if (
+            !!countryCodeFieldMessage &&
+            !!numberFieldMessage &&
+            !countryCodeAndNumberFieldMessage
+        ) {
+            throw new Error(
```

Comment:
In this design if both countryCode and numberField is invalid you should be using the countryCodeAndNumberFieldMessage-prop. We found that other components in FFE used this way to throw errors if props are used wrong. For instance if you use both ToolTip and Description for the RadioButtonInputGroup-component.

Is there a better way to do this? As this would result in runtime errors, and might be hard to catch during development.

Reply:
Yes, var i grunn det vi også tenkte, men prøvde å være konsekvent med de andre komponentene her, men da fjerner vi denne :+1:

Reply:
Vi la det inn som en console.error.